src/oic/utils/sdb.py

```diff
@@ -130,6 +130,18 @@ def get_type(self, token):
     def expires_at(self):
         return utc_time_sans_frac() + self.lifetime
 
+    def is_expired(self, when=0):
+        if not when:
+            now = time.time()
```

Comment:
What bothers me is the usage of two different time based utilities.

`now` uses `time` and `eat` uses `datetime` It would be better to use only one of them.